frontend/public/components/utils/status-box.tsx

```diff
@@ -54,7 +54,7 @@ const Data: React.FC<DataProps> = ({EmptyMsg, label, data, children}) => {
       {EmptyMsg ? <EmptyMsg /> : <EmptyBox label={label} />}
     </div>;
   }
-  return <div className="loading-box loading-box__loaded">{children}</div>;
```

Comment:
We key on the class in the tests in several places, so integration tests will need to be updated.

Reply:
I see...
The node really has no use other than it's being used in tests.
What about if we move the node to be part of the page but not part of the content hierarchy for now? That way it doesn't interfere with the layout of any child nodes.

Reply:
Yeah, I wouldn't be against this. It's only in a handful of places, though. We might just add `data-test-` attributes where we need them.

```
❯ rg loading-box__loaded
frontend/integration-tests/views/olm-catalog.view.ts
9:export const isLoaded = () => browser.wait(until.presenceOf($('.loading-box__loaded')), 10000).then(() => browser.sleep(500));

frontend/integration-tests/views/crud.view.ts
19:export const untilLoadingBoxLoaded = until.presenceOf($('.loading-box__loaded'));
122:export const modalAnnotationsLink = $('.loading-box__loaded').element(by.partialButtonText('Annotation'));

frontend/integration-tests/tests/olm/single-installmode.scenario.ts
91:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);
128:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);
174:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);
220:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);

frontend/integration-tests/tests/olm/global-installmode.scenario.ts
87:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);
122:    await browser.wait(until.presenceOf($('.loading-box__loaded')), 5000);

frontend/public/components/utils/status-box.tsx
53:    return <div className="loading-box loading-box__loaded">
57:  return <div className="loading-box loading-box__loaded">{children}</div>;
```

Reply:
I fixed the one failing test case related to the annotations and left the rest alone for now.